src/main/kotlin/com/linker/linkerapi/rental/enums/RentalStatus.kt

```diff
@@ -0,0 +1,8 @@
+package com.linker.linkerapi.rental.enums
+
+enum class RentalStatus {
+    REQUESTED,       // 대여 신청
+    PREPARING,       // 대여 준비 완료
```

Comment:
```suggestion
    PREPARED
```

다른 곳도 다 ED 형태이니 이것도 그렇게 통일하는게 좋아보입니다.